docs/fundamentals/diagnostics/runtime-contention-events.md

```diff
@@ -1,58 +1,52 @@
 ---
-title: "Thread Contention Events"
+title: "Monitor lock contention runtime events"
 description: See ETW events that collect information specific to the monitor lock contentions.
 ms.date: "09/27/2020"
 ms.topic: reference
 helpviewer_keywords:
-  - "thread contention events" [.NET Core]"
+  - "monitor lock contention events" [.NET Core]"
   - "ETW, EventPipe, LTTng contention events (CoreCLR)"
-ms.assetid: 167a4459-bb6e-476c-9046-7920880f2bb5
 ---
 
-# Contention Events
+# .NET runtime contention events
 
-These runtime events capture information about monitor lock contentions such as with Monitor.Enter or the C# lock keyword.
+These runtime events capture information about monitor lock contentions such as with `Monitor.Enter` or the C# lock keyword. For more information about how to use these events for diagnostic purposes, see [logging and tracing .NET applications](../../core/diagnostics/logging-tracing.md)
 
-This category consists of the following events:
+## ContentionStart_V1 event
 
-- [ContentionStart_V1](#contentionstart_v1-event)
-- [ContentionStop_V1](#contentionstop_v1-event)
+This event is emitted at the start of a managed thread contention.
```

Comment:
```suggestion
This event is emitted at the start of a monitor lock contention.
```

There are a few more references to "managed thread contention" that could also be updated